### ISSUE TITLE
browsing Dashboard connects to multiple external domains

### ISSUE THREAD

**Issue by steven-omaha** [traefik/traefik#7699]
### Do you want to request a *feature* or report a *bug*?
Bug

### What did you do?
I have been using traefik for many months. Today while checking `traefik.mydomain.one` I noticed ublock origin is blocking some external domains. Upon investigating, I found browsing the api@internal page sends requests to multiple external domains. This is a privacy issue.

Is there a way to disable all requests to external domains / external resources, while keeping access to api@internal?

### What did you expect to see?
Browsing `api@internal` connects to `traefik.mydomain.one`, and nothing else.

### What did you see instead?
Browsing `api@internal` connects to, besides my domain, multiple external domains, like vercel-dns.com, googletagmanager.com, metomic.io and cloudfront.net. This is a privacy issue.

Logfile from ublock origin when opening traefik.mydomain.one at the very end of the issue.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      2.3.6
Codename:     picodon
Go version:   go1.15.6
Built:        2020-12-17T16:34:27Z
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

Debian buster, docker-compose, all configured via docker cli args and docker labels.

```yaml
version: "3.6"

services:
  traefik:
    image: "traefik:v2.3"
    container_name: "traefik"
    domainname: "mydomain.one"
    restart: always
    command:
      - "--log.level=INFO"
      - "--api=true"
      - "--api.dashboard=true"
      - "--providers.docker=true"
      - "--providers.docker.network=traefik_proxy"  # use this network to connect to other containers
      - "--providers.docker.exposedbydefault=false"
      - "--entrypoints.web.address=:80"
      - "--entrypoints.websecure.address=:443"
      - "--certificatesresolvers.myresolver.acme.tlschallenge=true"
      - "--certificatesresolvers.myresolver.acme.email=webmaster@mydomain.one"
      - "--certificatesresolvers.myresolver.acme.storage=/letsencrypt/acme.json"
#     - "--certificatesResolvers.myresolver.acme.caServer=https://acme-staging-v02.api.letsencrypt.org/directory"
      - "--providers.file.directory=/rules"
      - "--providers.file.watch=true"
      - "--accesslog=true"
      - "--accesslog.format=json"
      - "--accessLog.filters.statusCodes=400-499"
    ports:
      - "80:80"
      - "443:443"
    volumes:
      - "./letsencrypt:/letsencrypt"
      - "./rules:/rules:ro"
      - "/var/run/docker.sock:/var/run/docker.sock:ro"
    networks:
      - traefik_proxy
      - bridge
    labels:
      - "traefik.enable=true"

      - "traefik.http.routers.api-plain.rule=Host(`traefik.mydomain.one`)"
      - "traefik.http.routers.api-plain.entrypoints=web"
      - "traefik.http.routers.api-plain.service=api@internal"
      - "traefik.http.routers.api-plain.middlewares=securityheaders-all@docker,permissionspolicy-default@docker"

      - "traefik.http.routers.api-secure.rule=Host(`traefik.mydomain.one`)"
      - "traefik.http.routers.api-secure.entrypoints=websecure"
      - "traefik.http.routers.api-secure.service=api@internal"
      - "traefik.http.routers.api-secure.tls.certresolver=myresolver"
      - "traefik.http.routers.api-secure.middlewares=securityheaders-all@docker,permissionspolicy-default@docker,authenticate@docker"

      - "traefik.http.middlewares.authenticate.basicauth.users=[REDACTED]

      - "traefik.http.middlewares.securityheaders-default.headers.framedeny=true"
      - "traefik.http.middlewares.securityheaders-default.headers.customFrameOptionsValue=SAMEORIGIN"
      - "traefik.http.middlewares.securityheaders-default.headers.contentTypeNoSniff=true"
      - "traefik.http.middlewares.securityheaders-default.headers.sslredirect=true"
      - "traefik.http.middlewares.securityheaders-default.headers.stsPreload=true"
      - "traefik.http.middlewares.securityheaders-default.headers.stsIncludeSubdomains=true"
      - "traefik.http.middlewares.securityheaders-default.headers.stsSeconds=31536000"
      - "traefik.http.middlewares.securityheaders-default.headers.forceSTSHeader=true"

      - "traefik.http.middlewares.securityheaders-onlystrict.headers.contentSecurityPolicy=script-src 'self'"
      - "traefik.http.middlewares.securityheaders-onlystrict.headers.referrerPolicy=same-origin"

      - "traefik.http.middlewares.securityheaders-all.chain.middlewares=securityheaders-default@docker,securityheaders-onlystrict@docker"

      - "traefik.http.middlewares.permissionspolicy-lenient.headers.customResponseHeaders.Permissions-Policy=geolocation 'none'; microphone 'none'; autoplay 'self'; camera 'none'; display-capture 'none'; midi 'none'; payment 'none'; fullscreen 'self'"
      - "traefik.http.middlewares.permissionspolicy-default.headers.customResponseHeaders.Permissions-Policy=geolocation 'none'; microphone 'none'; autoplay 'none'; camera 'none'; display-capture 'none'; midi 'none'; payment 'none'; fullscreen 'none'"

    logging:
      options:
        max-size: '12m'
        max-file: '5'
      driver: json-file

networks:
  traefik_proxy:
    external:
      name: traefik_proxy
  bridge:
    driver: bridge
```

Logfile from ublock origin:
```
+17			traefik.mydomain.one	1	xhr	https://traefik.mydomain.one/api/overview
+12			traefik.mydomain.one	1	xhr	https://traefik.mydomain.one/api/overview
+7			traefik.mydomain.one	1	xhr	https://traefik.mydomain.one/api/overview
+2			pilot.traefik.io	3,3	script	https://dd166b26j8giv.cloudfront.net/	aliasURL=https://consent-manager.metomic.io/Placeholders.2dc780d2504eb6b9dd1c.js
+2			pilot.traefik.io	3,3	script	https://consent-manager.metomic.io/Placeholders.2dc780d2504eb6b9dd1c.js
+2			pilot.traefik.io	3,3	script	https://dd166b26j8giv.cloudfront.net/	aliasURL=https://consent-manager.metomic.io/vendors~Placeholders.263685c3e446d79cad35.js
+2			pilot.traefik.io	3,3	script	https://consent-manager.metomic.io/vendors~Placeholders.263685c3e446d79cad35.js
+2			pilot.traefik.io	3,3	script	https://dd166b26j8giv.cloudfront.net/	aliasURL=https://consent-manager.metomic.io/ConsentManager.12805195a16f464d55f2.js
+2			pilot.traefik.io	3,3	script	https://consent-manager.metomic.io/ConsentManager.12805195a16f464d55f2.js
+2			pilot.traefik.io	3,3	script	https://dd166b26j8giv.cloudfront.net/	aliasURL=https://consent-manager.metomic.io/common~modules.2186f2b8678853f5838e.js
+2			pilot.traefik.io	3,3	script	https://consent-manager.metomic.io/common~modules.2186f2b8678853f5838e.js
+2			pilot.traefik.io	3,3	script	https://dd166b26j8giv.cloudfront.net/	aliasURL=https://consent-manager.metomic.io/vendors~modules.fce9fe08a731c0813aa3.js
+2			pilot.traefik.io	3,3	script	https://consent-manager.metomic.io/vendors~modules.fce9fe08a731c0813aa3.js
+2			pilot.traefik.io	3,3	script	https://dd166b26j8giv.cloudfront.net/	aliasURL=https://consent-manager.metomic.io/error-reporting.js
+2			pilot.traefik.io	3,3	script	https://consent-manager.metomic.io/error-reporting.js
+2			pilot.traefik.io	3,3	script	https://dd166b26j8giv.cloudfront.net/	aliasURL=https://consent-manager.metomic.io/bundle.js
+2			pilot.traefik.io	3,3	script	https://consent-manager.metomic.io/bundle.js
+2			pilot.traefik.io	3,3	xhr	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/api/me
+2			pilot.traefik.io	3,1	xhr	https://pilot.traefik.io/api/me
+2	googletagmanager.com/gtm.js	<<	pilot.traefik.io	3,3	script	https://www.googletagmanager.com/gtm.js?id=GTM-P26SFBV
+2	||googletagmanager.com/gtm.js$script,redirect-rule=googletagmanager.com/gtm.js	--	pilot.traefik.io	3,3	script	https://www.googletagmanager.com/gtm.js?id=GTM-P26SFBV
+2	||googletagmanager.com^	--	pilot.traefik.io	3,3	script	https://www.googletagmanager.com/gtm.js?id=GTM-P26SFBV
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/XjY766lT4RV3tw3gSFufY/_ssgManifest.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/XjY766lT4RV3tw3gSFufY/_ssgManifest.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/XjY766lT4RV3tw3gSFufY/_buildManifest.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/XjY766lT4RV3tw3gSFufY/_buildManifest.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/chunks/pages/partials/auth-state-52816d9bc7035aa752c9.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/chunks/pages/partials/auth-state-52816d9bc7035aa752c9.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/chunks/7d1e5b98f55f8a2abec05f04f05cc95910b6c796.60b5da60afe49bf7a1bf.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/chunks/7d1e5b98f55f8a2abec05f04f05cc95910b6c796.60b5da60afe49bf7a1bf.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/chunks/pages/_app-8ed8a729516a378cd90a.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/chunks/pages/_app-8ed8a729516a378cd90a.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/chunks/b1542cc239bf56c8a14913c0a930efe682b5cf86.f227a077a3bb6c15719b.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/chunks/b1542cc239bf56c8a14913c0a930efe682b5cf86.f227a077a3bb6c15719b.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/chunks/c6be70595d179d1678e770949a8b9fddcf4fb531.69c5b5e974b4c67783f2.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/chunks/c6be70595d179d1678e770949a8b9fddcf4fb531.69c5b5e974b4c67783f2.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/chunks/a29eaf6c8f8eade780bbff3081df702f646777f0.d34a5fb3ce8f7567804f.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/chunks/a29eaf6c8f8eade780bbff3081df702f646777f0.d34a5fb3ce8f7567804f.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/chunks/04ffad4b9a6532fae7b15c74e43a5d70f7c56e9d.9ebd515493c9d1ff8c23.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/chunks/04ffad4b9a6532fae7b15c74e43a5d70f7c56e9d.9ebd515493c9d1ff8c23.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/chunks/c353b068.3d554dc21415e76e9f43.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/chunks/c353b068.3d554dc21415e76e9f43.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/chunks/b46a2385.dbaa3be89c9ef697edaa.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/chunks/b46a2385.dbaa3be89c9ef697edaa.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/chunks/7c8a5e5b.9f5d902f71ae76e75add.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/chunks/7c8a5e5b.9f5d902f71ae76e75add.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/chunks/8febc35a.2e5dadbc45af5cd4253b.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/chunks/8febc35a.2e5dadbc45af5cd4253b.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/chunks/framework.f30ca4b53ab2846e3a57.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/chunks/framework.f30ca4b53ab2846e3a57.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/chunks/webpack-e067438c4cf4ef2ef178.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/chunks/webpack-e067438c4cf4ef2ef178.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/_next/static/chunks/main-fd8bcd7f7e37c2e88ed1.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/_next/static/chunks/main-fd8bcd7f7e37c2e88ed1.js
+2			pilot.traefik.io	3,3	script	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/iframeResizer.contentWindow.min.js
+2			pilot.traefik.io	3,1	script	https://pilot.traefik.io/iframeResizer.contentWindow.min.js
+2			pilot.traefik.io	3,3	script	https://dd166b26j8giv.cloudfront.net/	aliasURL=https://consent-manager.metomic.io/embed.js
+2			pilot.traefik.io	3,3	script	https://consent-manager.metomic.io/embed.js
+2			traefik.mydomain.one	1	xhr	https://traefik.mydomain.one/api/version
+2			traefik.mydomain.one	1	image	https://traefik.mydomain.one/dashboard/statics/icons/favicon-96x96.png
+2			traefik.mydomain.one	1	image	https://traefik.mydomain.one/dashboard/statics/icons/apple-icon-180x180.png
+2			traefik.mydomain.one	3	frame	https://cname.vercel-dns.com/	aliasURL=https://pilot.traefik.io/partials/auth-state?authRedirectUrl=https%3A%2F%2Ftraefik.mydomain.one%2Fdashboard%2F%23%2F%3Fplatform%3D%2F&instanceInfos=%7B%22Version%22%3A%222.3.6%22%2C%22Codename%22%3A%22picodon%22%2C%22startDate%22%3A%222020-12-19T18%3A53%3A54.482241387Z%22%7D
+2			traefik.mydomain.one	3	frame	https://pilot.traefik.io/partials/auth-state?authRedirectUrl=https%3A%2F%2Ftraefik.mydomain.one%2Fdashboard%2F%23%2F%3Fplatform%3D%2F&instanceInfos=%7B%22Version%22%3A%222.3.6%22%2C%22Codename%22%3A%22picodon%22%2C%22startDate%22%3A%222020-12-19T18%3A53%3A54.482241387Z%22%7D
+2			traefik.mydomain.one	1	xhr	https://traefik.mydomain.one/api/overview
+2			traefik.mydomain.one	1	xhr	https://traefik.mydomain.one/api/entrypoints
+2			traefik.mydomain.one	1	xhr	https://traefik.mydomain.one/api/version
+2			traefik.mydomain.one	1	xhr	https://traefik.mydomain.one/api/version
+2			traefik.mydomain.one	1	script	https://traefik.mydomain.one/dashboard/js/d2880142.e9acfed0.js
+2			traefik.mydomain.one	1	script	https://traefik.mydomain.one/dashboard/js/b43aa09a.9233cb57.js
+2			traefik.mydomain.one	1	script	https://traefik.mydomain.one/dashboard/js/vendor.82e001d5.js
+2			traefik.mydomain.one	1	script	https://traefik.mydomain.one/dashboard/js/runtime.4ea52ed1.js
+2			traefik.mydomain.one	1	script	https://traefik.mydomain.one/dashboard/js/app.8df9dc1d.js
+1			traefik.mydomain.one	1	doc	https://traefik.mydomain.one/dashboard/
+1	https://traefik.mydomain.one/dashboard/
+1			traefik.mydomain.one	1	doc	https://traefik.mydomain.one/
+1	https://traefik.mydomain.one/
+0			traefik.mydomain.one	1	xhr	https://traefik.mydomain.one/api/overview

```

**Comment by SebDanielsson** [comment]
I'm noticed the same. Why should Traefik connect to Google for running a reverse proxy?

**Comment by gurre** [comment]
Traefik Pilot is an anti-feature that should be opt-in, or even better, a separate service.

```
fonts.googleapis.com
pilot.traefik.io
apipub.metomic.io
consent-manager.metomic.io
```

**Comment by meyca** [comment]
related to #7360

**Comment by ldez** [comment]
Closed by #7994.
